services/token-reward-service/.env

```
@@ -1,16 +1,50 @@
 DB_USER=postgres
 DB_PASSWORD=password
 DB_HOST=localhost
 DB_PORT=5433
 DB_NAME=course_db
 
+# =====================================================================
+# BLOCKCHAIN CONFIGURATION (Optional)
+# =====================================================================
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
-ACCOUNT_PRIVATE_KEY=0x22821a351140957d1c69a151ac67d691c8f2671991d1479ac80bdfa5723bf291
+ACCOUNT_PRIVATE_KEY=0x4fcea8c0f36ac7fd8f1033cae252d941176a4d530913150da6e5239129c5372d
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)